File: BOJ/01000/1919.cpp
#include <bits/stdc++.h>
using namespace std;

int cnt[26], ret;
string a, b;

int main() {
  ios_base::sync_with_stdio(false);
  cin.tie(NULL);
  cout.tie(NULL);

  cin >> a >> b;

  for (char c : a) cnt[c - 'a']++;
  for (char c : b) cnt[c - 'a']--;

  for (int it : cnt) {
    if (it != 0) ret += abs(it);
  }

  cout << ret;

  return 0;
}

File: BOJ/01000/1919.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "../input.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

let ret = 0;
const cnt = Array.from({ length: 26 }).fill(0);
const [a, b] = input.map((v) => v.split(""));

a.forEach((v) => {
  cnt[v.charCodeAt() - "a".charCodeAt()]++;
});

b.forEach((v) => {
  cnt[v.charCodeAt() - "a".charCodeAt()]--;
});

cnt.forEach((v) => {
  if (v !== 0) ret += Math.abs(v);
});

console.log(ret);
